tooling/api/src/main/java/org/jboss/windup/tooling/ProgressMonitorAdapter.java

```diff
@@ -0,0 +1,65 @@
+package org.jboss.windup.tooling;
+
+import java.util.logging.LogRecord;
+
+import org.jboss.windup.tooling.IProgressMonitorAdapter;
+import org.jboss.windup.tooling.WindupToolingProgressMonitor;
+
+public class ProgressMonitorAdapter implements IProgressMonitorAdapter, WindupToolingProgressMonitor 
```

Comment:
It's not clear for me the use for this class. As it only has the methods from the interface, and it becomes an adapter of the same interface..... So, the client of this class could not simply call the class implementing the IProgressMonitorAdapter and not use this adapter ?

Reply:
Correct. This cleans things up. The adapter delegates. WindupToolingProgressMonitor callbacks are then delegated to the ToolingModeRunner.